rangeserver/src/server.rs

```diff
@@ -22,6 +24,33 @@ use crate::{
 use flatbuf::rangeserver_flatbuffers::range_server::TransactionInfo as FlatbufTransactionInfo;
 use flatbuf::rangeserver_flatbuffers::range_server::*;
 
+use proto::rangeserver::range_server_server::{RangeServer, RangeServerServer};
+use proto::rangeserver::{PrefetchRequest, PrefetchResponse};
+
+pub mod rangeserver {
+    include!("../../proto/target/rangeserver/rangeserver.rs");
+}
+
+#[derive(Debug, Default)]
+pub struct RSPrefetch {}
```

Comment:
I'd just call that ProtoServer or something similar. Is there a reason this needs to be pub btw?